ci/ci.sh

```diff
@@ -543,17 +533,22 @@ lint_annotations() {
 }
 
 lint_bazel() {
-  # Run buildifier without affecting external environment variables
-  (
-    mkdir -p -- "${GOPATH}"
-    export PATH="${GOPATH}/bin:${GOROOT}/bin:${PATH}"
+  if [[ ! "${OSTYPE}" =~ ^linux ]]; then
+    echo "Bazel lint not supported on non-linux systems."
+    exit 1
```

Comment:
is this function currently called on non-supported system?

Reply:
I searched and no. this is only called on ci, and ci only runs lints on linux `x86_64`.

People can still run the bazel lint directly, which assumes that `buildifier` is already installed on the system.